src/Commands/CommandFactory.cs

```diff
@@ -348,14 +348,14 @@ private void RegisterServiceBusCommands()
         _rootGroup.AddSubGroup(serviceBus);
 
         var queue = new CommandGroup("queue", "Queue operations - Commands for using Azure Service Bus queues.");
-        queue.AddCommand("peek", new ServiceBus.Queue.QueuePeekCommand());
+        // queue.AddCommand("peek", new ServiceBus.Queue.QueuePeekCommand());
         queue.AddCommand("details", new ServiceBus.Queue.QueueDetailsCommand());
 
         var topic = new CommandGroup("topic", "Topic operations - Commands for using Azure Service Bus topics and subscriptions.");
         topic.AddCommand("details", new ServiceBus.Topic.TopicDetailsCommand());
 
         var subscription = new CommandGroup("subscription", "Subscription operations - Commands for using subscriptions within a Service Bus topic.");
-        subscription.AddCommand("peek", new ServiceBus.Topic.SubscriptionPeekCommand());
+        // subscription.AddCommand("peek", new ServiceBus.Topic.SubscriptionPeekCommand());
```

Comment:
Remove?

Reply:
Yeah. At the moment we don't know how to show the body for a message... It is an opaque set of bytes. So it could be an image, it could be a string, could be etc..

Reply:
I'm afraid if we assume it is a string, it could be random garbage data when decoded.